images/manageiq-base/Dockerfile

```diff
@@ -31,13 +31,17 @@ LABEL name="manageiq-base" \
       io.openshift.tags="ManageIQ,miq,manageiq"
 
 COPY rpms/* /tmp/rpms/
+COPY s390x/*.rpm /tmp/rpms/
+RUN if [ ${ARCH} = "s390x" ]; then dnf -y install /tmp/rpms/python3-ovirt-engine-sdk4-4.4.10-1.el8.s390x.rpm ; fi
 COPY container-assets/create_local_yum_repo.sh /
 
 RUN curl -L https://releases.ansible.com/ansible-runner/ansible-runner.el8.repo > /etc/yum.repos.d/ansible-runner.repo
 
-RUN dnf -y --disableplugin=subscription-manager install \
+RUN if [ ${ARCH} != "s390x" ] ; then dnf -y --disableplugin=subscription-manager install \
       http://mirror.centos.org/centos/8.3.2011/BaseOS/${ARCH}/os/Packages/centos-linux-repos-8-2.el8.noarch.rpm \
-      http://mirror.centos.org/centos/8.3.2011/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm \
+      http://mirror.centos.org/centos/8.3.2011/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm \ 
+      https://resources.ovirt.org/pub/yum-repo/ovirt-release44.rpm; fi && \
```

Comment:
This line already exists a few lines down, so we shouldn't need it inside the conditional as well, right?